db-init/seed.sql

```diff
@@ -0,0 +1,24 @@
+-- Seed log records table
+INSERT INTO log_records (
+  employee_id, 
+  resident_first_name, 
+  resident_last_name, 
+  datetime, 
+  flagged, 
+  attn_to, 
+  note, 
+  tags, 
+  building ) 
+
+SELECT
```

Comment:
would we able to do something similar to set up residents and a few dummy accounts in the db? It would definitely help set up a testing env fairly quickly and we can include an admin account here and in Firebase (so we don't have to keep creating accounts on fresh runs of the containers)

Reply:
Yes